xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/service/BittrexTradeServiceRaw.java

```diff
@@ -64,7 +64,7 @@ public String placeBittrexLimitOrder(LimitOrder limitOrder, TimeInForce type) th
   }
 
   public String placeBittrexMarketOrder(MarketOrder marketOrder) throws IOException {
-    return placeBittrexMarketOrder(marketOrder, TimeInForce.GOOD_TIL_CANCELLED);
+    return placeBittrexMarketOrder(marketOrder, TimeInForce.IMMEDIATE_OR_CANCEL);
```

Comment:
is this correct/expected behavior on a market order?

Reply:
The previous code returned 409 http error. GOOD_TIL_CANCELLED is used only for LIMIT order and not MARKET order. More in the comment.